webpack/components/TemplateSyncResult/components/SyncResultList.js

```diff
@@ -2,13 +2,19 @@ import React from 'react';
 import { ListView } from 'patternfly-react';
 import PropTypes from 'prop-types';
 
-import Pagination from 'foremanReact/components/Pagination/PaginationWrapper';
+import Pagination from 'foremanReact/components/Pagination';
 
 import SyncedTemplate from './SyncedTemplate';
 import { templatesPage } from '../TemplateSyncResultHelpers';
 import ListViewHeader from './ListViewHeader';
 
-const SyncResultList = ({ pagination, pageChange, templates, editPaths }) => (
+const SyncResultList = ({
+  pagination,
+  pagination: { page, perPage },
```

Comment:
Well I thought we could just camalize in the reducer, but now this will be wrong, won't it?
This should now be
```suggestion
  pagination: { page, perPage: per_page },
```

Reply:
tested, should work now :)
at first, I tried to camel case the pagination object, but it ended with some errors 🤔 

Also, I created https://github.com/theforeman/foreman/pull/9097 to fix the missing variant, so the page selection dropdown  will be open correctly